pyomo/core/tests/unit/test_model.py

```diff
@@ -267,6 +267,14 @@ def e_rule(model, i):
             join(currdir,"solve1b.out"), join(currdir,"solve1b.txt"),
             tolerance=1e-4)
 
+        # test store_to() function with skip_stale_vars=True
```

Comment:
You're on the right track with this test but I think it could be improved by moving it to its own test function and using `store_to` on a model that has both `stale=True` and `stale=False` variables (tested with both `skip_stale_vars=True` and `skip_stale_vars=False`). If there is a way to query the results object to verify whether or not it contains variables you know to be stale that would be a more robust test than the file baselines.

Reply:
Hey @blnicho , I added a test function for `store_to` with `skip_stale_vars=True` and `skip_stale_vars=False`. Let me know if it needs to further modified. Thanks.